src/index.ts

```diff
@@ -10,60 +10,57 @@
  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
  */
-import { Field } from './types';
+
 import { Event } from './event';
-import { LATITUDE_INDEX, LONGITUDE_INDEX, EVENT_STRUCTURE } from './structure';
+import { Field } from './types';
+import { EVENT_STRUCTURE, LATITUDE_INDEX, LONGITUDE_INDEX } from './structure';
 
 /**
- * Convert an aray of field values to a JSON whose keys are the field names
+ * Convert an aray of field values to a JSON whose keys are the field names.
  *
- * @param event Array of values for the event
- * @return JSON for the event
+ * @param event Array of values for the event.
+ * @return JSON for the event.
  */
 function jsonifyGoodEvent(event: string[]): Event {
   if (event.length !== Object.keys(EVENT_STRUCTURE).length) {
     throw new TypeError('Wrong event fields number.');
   }
 
-  const errors = [] as string[],
-    output = Object.keys(EVENT_STRUCTURE)
-    .reduce((output: Field[], key, index) => {
-      if (event[index]) {
-        try {
-          output.push(...EVENT_STRUCTURE[key](key, event[index]));
-        } catch (e) {
-          errors.push(e.message);
+  const errors = [] as string[];
+  const output = Object.keys(EVENT_STRUCTURE)
+    .reduce(
+      (output, key, index) => {
```

Comment:
This line fails linting locally for me due to the new line after `.reduce(` and the new line at the bottom of the function before `[] as Field[],`. Running --fix reshuffles this section a little and fixes linting locally.
I'm not entirely sure why this passes on Travis and fails locally however, I have tested the altered version on Travis and it also passes: https://travis-ci.org/snowplow-incubator/snowplow-js-analytics-sdk/builds/617177080